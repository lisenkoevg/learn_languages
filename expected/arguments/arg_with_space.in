#args 1 "ab cd" 2 
