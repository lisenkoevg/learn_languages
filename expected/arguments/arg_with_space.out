arg1: 1
arg2: ab cd
arg3: 2
